wear/src/main/java/com/android/gpstest/wear/MainActivity.kt

```diff
@@ -162,7 +167,7 @@ class MainActivity : ComponentActivity() {
 
 @OptIn(ExperimentalCoroutinesApi::class)
 @Composable
-fun WearApp(signalInfoViewModel: SignalInfoViewModel) {
```

Comment:
Could you please rename `WearApp` to `StatusScreen` and move it to it's own file?